### ISSUE TITLE
getPort.makeRange clashing when running parallel tests

### ISSUE THREAD

**Issue by HunorMarton** [sindresorhus/get-port#37]
I'm using `ava` to set up multiple CDCT tests starting with this:
```
const test = require('ava')
const getPort = require('get-port')
const { Pact } = require('@pact-foundation/pact')

...

test.before('async() => {
  port = await getPort()
  console.log(`Setting up port ${port}`)

  provider = new Pact({
    port,
    ...
  })

  await provider.setup()
})

...
```

Works great, tests are running in parallel and different ports are used:
```
Setting up on port 64873
Setting up on port 64874
```

But Pact gives me a warning saying I'm using a port outside it's recommended range:
```
WARN: pact-node@10.2.2/90509 on MBP.localdomain: Like a Boss, you used a port outside of the recommended range (1024 to 49151); I too like to live dangerously.
```

While it's just a warning, it's still not nice, so I change it to this:
```
test.before('async() => {
  port = await getPort({ port: getPort.makeRange(1024, 49151) })
  console.log(`Setting up port ${port}`)
  ...
})
```
But then both end up to be on the same port and my tests fail
```
Setting up on port 1024
Setting up on port 1024
```

Could the `makeRange` be randomized the same way as the default behaviour is?

**Comment by sindresorhus** [comment]
Duplicate of #23

**Comment by sindresorhus** [comment]
The in-process race condition you're experiencing was fixed by https://github.com/sindresorhus/get-port/commit/73e21d8c2f9ad2d4eb6bff0503292f44014b8bfd.
